docker-compose.yml

```diff
@@ -1,28 +1,32 @@
 version: '2'
 services:
   web:
-    image: fpfis/httpd-php-dev:7.3
+    image: registry.fpfis.eu/fpfis/httpd-php:7.3-dev
```

Comment:
We didn't update the images as the ones behind registry.fpfis.eu/fpfis/ require a Gitlab account and this is a major problem for the opensource side of OpenEuropa. Everybody should be able to setup the component locally. So no images in docker-compose.yml should be touched.